c/default.nix

```
@@ -0,0 +1,13 @@
+{ stdenv, gnumake }:
+stdenv.mkDerivation {
+  name = "hello";
+
+  src = ./.;
+  nativeBuildInputs = [ gnumake ];
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
+    install -D hello $out/bin/hello --mode 0755
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall

    install -Dm755 hello $out/bin/hello
    
    runHook postInstall
  '';
```

this is also another reason to use cmake or meson: this phase wouldn't be required